src/App.tsx

```diff
@@ -1,14 +1,69 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useState, useEffect, useCallback } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
+import { getTodos, getUser } from './api';
+import { User } from './types/User';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [selectedTodo, setSelectedTodo] = useState<Todo | null>(null);
+
+  const [selectedUser, setSelectedUser] = useState<User | null>(null);
+
+  const [isModalOpen, setModalOpen] = useState(false);
+
+  const [sort, setSort] = useState('all');
+  const [query, setQuery] = useState('');
+
+  const [isLoading, setIsLoading] = useState(true);
+  const [hasError, sethasError] = useState(false);
+
+  const loadTodos = useCallback(async () => {
+    try {
+      const todoList = await getTodos();
+
+      setIsLoading(false);
+      setTodos(todoList);
+    } catch {
+      sethasError(true);
```

Comment:
You have used 'sethasError' function with a lowercase 'h'. It should be 'setHasError' following camelCase naming convention.